Content.Server/Power/SMES/SmesSystem.cs

```diff
@@ -0,0 +1,80 @@
+using Content.Server.Power.Components;
+using Content.Server.Power.EntitySystems;
+using Content.Shared.Power;
+using Content.Shared.Rounding;
+using Content.Shared.SMES;
+using JetBrains.Annotations;
+using Robust.Shared.Timing;
+
+namespace Content.Server.Power.SMES;
+
+[UsedImplicitly]
+internal sealed class SmesSystem : EntitySystem
+{
+    private static TimeSpan VisualsChangeDelay = TimeSpan.FromSeconds(1);
```

Comment:
I missed this before approving but storing information in the system is very discouraged, what i meant from changing the static field in the component was that it should stay in the component just without being static

Reply:
Moved back to component and made non-static (though, im not sure why it should be non-static given that it is never changed!)